OpenRA.Mods.Common/UtilityCommands/PngSheetExportMetadataCommand.cs

```diff
@@ -0,0 +1,41 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.IO;
+using OpenRA.FileFormats;
+
+namespace OpenRA.Mods.Common.UtilityCommands
+{
+	public class PngSheetExportMetadataCommand : IUtilityCommand
+	{
+		string IUtilityCommand.Name { get { return "--PngSheetExport"; } }
+
+		bool IUtilityCommand.ValidateArguments(string[] args)
+		{
+			return args.Length == 3;
+		}
+
+		[Desc("PNGFILE YAMLFILE", "Export png metadata to yaml")]
```

Comment:
For conversion commands we typically use the same filename as the input but change the extension (so calling on bunny.png would write bunny.yaml)

Reply:
Need to remove the YAMLFILE here